src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/BfdSpec.groovy

```diff
@@ -71,6 +79,25 @@ class BfdSpec extends HealthCheckSpecification {
             }
         }
 
+        and: "BFD logical ports are created"
+        def srcSwLogPort = isl.srcPort + bfdLogPortDelta
+        def dstSwLogPort = isl.dstPort + bfdLogPortDelta
+        def srcSwitchAddress = northbound.getSwitch(isl.srcSwitch.dpId).address
+        def dstSwitchAddress = northbound.getSwitch(isl.dstSwitch.dpId).address
+        [isl.srcSwitch, isl.dstSwitch].each { sw ->
+            def swAddress = sw.dpId == isl.srcSwitch.dpId ? srcSwitchAddress : dstSwitchAddress
```

Comment:
`def swAddress = northbound.getSwitch(it.dpId).address` and remove L85-86

Reply:
I would like to leave it as is.
These variables are reused in L158 and L165
It allows us to reduce the number of requests to `northbound`.

Reply:
ah, didn't notice